### PR TITLE
og:description rework

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

**Comment by enygmate** [comment]
yes

File: wiki/resources/basics.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
+title: Basics
 sidebar_position: 1
+description: Basic ressources.
```
Comment (DyDestroyer1027) [inline wiki/resources/basics.md:4]: Typo: ressources -> resources

File: wiki/resources/moderation.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
+title: Moderation
 sidebar_position: 6
+description: Moderation ressources.
```
Comment (DyDestroyer1027) [inline wiki/resources/moderation.md:4]: ressources -> resources

File: wiki/resources/unofficial-resources.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
+title: Unofficial Resources
 sidebar_position: 9
+description: Unofficial ressources for discord.
```
Comment (DyDestroyer1027) [inline wiki/resources/unofficial-resources.md:4]: ressources -> resources

**Review by DyDestroyer1027 — CHANGES_REQUESTED** [review]
Several instances of ressources where it should be resources. I also feel the period at the end of short descriptions is unnecessary.

**Review by DyDestroyer1027 — APPROVED** [review]
Looks good!

**Pull request merged**